__tests__/kata6.joinNames.test.js

```diff
@@ -2,6 +2,25 @@ const { joinNames } = require('../src');
 
 describe('joinNames', () => {
   it('returns string of names, seperated by commas and an ampersand', () => {
-    
-  });
-});
+    const names1 = [{
+        name: 'Bart'
+  }, {
+        name: 'Lisa'
+  }, {
+        name: 'Maggie'
+  }]
+    const names2 = 
+        [{
+        name: 'Rob'
+  }, {
+        name: 'Bob'
+  }, {
+        name: 'Lob'
+  }, {
+        name: 'Stew'
+      }]
```

Comment:
Both of these objects are redundant as you don't use them in your assertion below, so they can be deleted.